apps/pie-microsite/src/assets/styles/components/_contentPage.scss

```diff
@@ -67,9 +67,25 @@
     padding-top: f.spacing(f);
     padding-bottom: f.spacing(j);
 
+    > h2 {
+        @include f.font-size('heading-m');
+    }
+
```

Comment:
this is being overridden by line 99 - that's why it's not correct on mobile :)
<img width="435" alt="Screenshot 2023-01-17 at 16 00 37" src="https://user-images.githubusercontent.com/16766185/212947928-a13875d1-de05-47f2-9f04-9600f3df5b52.png">